src/main/java/com/microsoft/graph/models/extensions/IBaseGraphServiceClient.java

```diff
@@ -1294,51 +1287,6 @@
      */
     ITeamsTemplateRequestBuilder teamsTemplates(final String id);
 
-    /**
-     * Gets the collection of LinkedResources objects
-     *
-     * @return the request builder for the collection of LinkedResources objects
-     */
-    ILinkedResourceCollectionRequestBuilder linkedResources();
```

Comment:
@MIchaelMainer this doesn't look normal to me, thoughts?

Reply:
@baywet these changes are expected. Take a look at the latest beta metadata capture diff to confirm: https://github.com/microsoftgraph/msgraph-metadata/commit/2b614bafa0ab6d8c835fd8409e8eac7cc7b1e3e6#diff-1f82c09660ae27359655006d82393127L27314